base/db/R/get.trait.data.R

```diff
@@ -42,6 +43,45 @@ get.trait.data <-
     PEcAn.logger::logger.severe('At least one pft in settings is missing its "outdir"')
   }
   
+  #check for flatfile path if present use it 
+  file_path <- input_file 
+  if(is.null(file_path)){
+    file.path <- pfts$file_path
+  }
+  use_flatfile <- !is.null(file_path) && file.exists(file_path)
```

Comment:
I suggest throwing an error if file_path is specified but the file doesn't exist, rather than revert to DB mode (which would be surprising to the user).